readme.md

```diff
@@ -6,6 +6,10 @@
 
 Tool Kit is modern developer tooling for FT.com repositories. It's fully modular, allowing repos that need different tooling to install separate plugins that work consistently together.
 
+Tool Kit has been created to enable the FT.com development workflow and only handles common tooling use cases that are required for most apps to work. Your application does not need to use Tool Kit for all of its tooling and tooling not supported by Tool Kit can be configured directly in your application.
+
+Tool Kit requires the minimum possible configuration for third party packages to work in the FT.com development workflow - additional configuration is left to you.
```

Comment:
**question**: do you mean to say that Tool Kit sets up the minimal configuration for third party packages to run? I can't really parse this sentence 😅

Reply:
Yes, that's right. I did feel that these sentences were a bit garbled! 

Would this be better:

```MD
Tool Kit has been created to enable the FT.com development workflow and only handles common tooling use cases that are required for most apps to work. Tool Kit sets up the minimal configuration for third party packages to run.

Your application does not need to use Tool Kit for all of its tooling and tooling not supported by Tool Kit can be configured directly in your application.
```

Reply:
Yes I think that's much clearer, thank you!